src/Tools/dotnet-user-secrets/test/SecretManagerTests.cs

```diff
@@ -149,19 +149,20 @@ public void SetSecrets(bool fromCurrentDirectory)
         }
 
         [Fact]
+        [QuarantinedTest]
         public void SetSecret_Update_Existing_Secret()
         {
             var projectPath = _fixture.GetTempSecretProject();
             var secretManager = CreateProgram();
 
-            secretManager.RunInternal("set", "secret1", "value1", "-p", projectPath);
+            secretManager.RunInternal("set", "secret1", "value1", "-p", projectPath, "--verbose");
```

Comment:
Why add the new `[QuarantinedTest]` and extra argument in a PR about renaming?

Reply:
Because the test showed as flaky over the weekend and it's easier to add it to the PR that's changing the name of the attribute than it is to open a PR with the wrong name, and then have to deal with conflicts.

Reply:
:ok: